README.md

```diff
@@ -111,14 +124,18 @@ TODO: look through discussion threads to check that this solves the objections.
 
 ### When to cache the webfonts
 
-It would be safest to pre-cache all of these fonts when a new major version of a
+It's safest to pre-cache all of these fonts when a new major version of a
 browser is installed, but this might waste valuable bandwidth and disk space for
 a font that a particular user never happens to need.
 
-I believe it's also safe to cache each font at the point where it's first used,
-as long as the cache never evicts fonts. This allows exactly one site to
-determine that a user has not visited any site that either uses the font or has
-tried to learn this fact about the user.
+Caching each font at the point where it's first used, if the cache never evicts
+fonts, allows exactly one site to determine that a user has not visited any site
+that either uses the font or has tried to learn this fact about the user. If
+exactly one attacker notices a sensitive set of sites using a font, and that set
+is large enough to make the font "common", and the attacker has logged-in users,
+they might be able to probe each user exactly once in order to distinguish users
+who've visited the sensitive set without polluting their sample with users who
+have previously been attacked.
```

Comment:
I think this is missing the angle that users can also segment or be segmented in ways that are not necessarily clear to a global observer. E.g., they might only be able to visit one site with such a font in their region of the world or some such.

Reply:
I rewrote this to try to be more precise. Is 57a5c11e better?